config/internal/apiserver/default/deployment.yaml.tmpl

```diff
@@ -19,6 +19,8 @@ spec:
         app: {{.APIServerDefaultResourceName}}
         component: data-science-pipelines
         dspa: {{.Name}}
+        configHash: {{.APIServerConfigHash}}
```

Comment:
I assume this is to make the pod restart when the sample config changes. Would an annotation work instead?

Reply:
yeah my preference is annotation as well